### PR TITLE
Logging

### PR DESCRIPTION
**Pull request by MaxNoe**
add the handlers also to the root file handler, set level of root to info

set level and handlers for both `custos` and `shifthelper` logger

Log to stdout by default

### CODE REVIEW

**Comment by MaxNoe** [comment]
This will fix the logging hopefully

File: shifthelper/__main__.py [hunk 1]
```diff
@@ -13,7 +13,7 @@
 from .checks import FactIntervalCheck
 from . import conditions
 
-config_logging(to_console=False)
+config_logging(to_console=True)
```
Comment (dneise) [inline shifthelper/__main__.py]: so we log to console by default? 
for testing only for forever?

File: shifthelper/conditions.py [hunk 2]
```diff
@@ -242,6 +242,7 @@ def is_no_dimctrl_server_available():
 
 @log_call_and_result
 def is_no_shift_at_the_moment():
+    '''no shift at the moment'''
```
Comment (dneise) [inline shifthelper/conditions.py]: this is an important bugfix, which has nothing to do with logging. 
So it is good to have it, but would have been good to at least mention it in the discussion of this PR, I guess.

File: shifthelper/db_cloner/__main__.py [hunk 3]
```diff
@@ -52,12 +47,27 @@ def users():
...
+    )
+    formatter = logging.Formatter(
+        fmt='%(asctime)s|%(name)s|%(levelname)s|%(module)s|%(lineno)d|%(message)s'
+    )
+    formatter.converter = time.gmtime  # use utc in log
+    for handler in (logging.FileHandler(logfile), logging.StreamHandler()):
+        handler.level = logging.INFO
+        handler.setFormatter(formatter)
+        log.addHandler(handler)
+
```
Comment (dneise) [inline shifthelper/db_cloner/__main__.py]: These new environment vars, which the user/co-developer can use to steer stuff should also be mentioned in the README, I guess.

File: shifthelper/db_cloner/__main__.py [hunk 4]
```diff
@@ -52,12 +47,27 @@ def users():
...
     db_in = create_db_connection(config["database"])
     db_out = create_db_connection(config["cloned_db"])
 
     while True:
         try:
-            log.info("clonging ...")
+            log.info("cloning ...")
```
Comment (dneise) [inline shifthelper/db_cloner/__main__.py]: no no ... I actually meant "clonging" :-D

File: shifthelper/logging.py [hunk 5]
```diff
@@ -39,16 +40,26 @@ def config_logging(to_console=False, level=logging.DEBUG):
...
+        log.setLevel(level)
+        log.addHandler(logfile_handler)
+        log.addHandler(text_logfile_handler)
+
+        if to_console:
+            log.addHandler(stream_handler)
 
-    log = logging.getLogger('shifthelper')
-    log.setLevel(level)
+    log = logging.getLogger('root')
```
Comment (dneise) [inline shifthelper/logging.py]: I think this is wrong. The way to get the root logger is: `getLogger()` or `getLogger('')`, 
```
In [5]: logging.getLogger('') == logging.getLogger()
Out[5]: True

In [6]: logging.getLogger('') == logging.getLogger('root')
Out[6]: False
```

Or did you actually want the logger named `root`, If so I didn't know we had a logger of that name.

File: shifthelper/__main__.py (hunk 1 above)
Comment (MaxNoe) [inline shifthelper/__main__.py]: Maybe for testing only, but I found it really helpful.

File: shifthelper/db_cloner/__main__.py (hunk 3 above)
Comment (MaxNoe) [inline shifthelper/db_cloner/__main__.py]: Yes

File: shifthelper/logging.py (hunk 5 above)
Comment (MaxNoe) [inline shifthelper/logging.py]: Fuck, this is really confusing:

```
In [1]: import logging

In [2]: l1 = logging.getLogger()

In [3]: l1.name
Out[3]: 'root'

In [4]: l2 = logging.getLogger('root')

In [5]: l2.name
Out[5]: 'root'
```

Good catch

File: shifthelper/db_cloner/__main__.py (hunk 3 above)
Comment (MaxNoe) [inline shifthelper/db_cloner/__main__.py]: done

**Pull request merged**